### DISCUSSION TITLE
Trying to find the code for transition wipe that occurs before and after a song

### DISCUSSION THREAD

**Discussion by KatazukiFX** [KadeArchive/Kade-Engine#2276] [Q&A] [answered]
I've been looking for this code for weeks now and I can't seem to find where it's located.

I want to find this code because I wish to apply different zoom values to specific stages. However, if I zoom too far out, the edges of the transition state's cutoff becomes visible. I'd like to know where the code for this transition state is located so I can expand it more to accommodate for a zoomed out stage.

![](https://cdn.discordapp.com/attachments/830663948088508436/884651118045581323/unknown.png)

Thanks much!

**Comment by isakube** [comment, marked as answer]
MusicBeatState is an FlxUIState, which can have a Transitionable Effect going in and out of the state. In titlestate, the thingy is initialized somewhere

**Reply by KatazukiFX** [reply]
Oh, I believe I found it in titlestate.

![image](https://user-images.githubusercontent.com/87513619/134753363-f96dfa80-de34-4166-964a-93e92296861d.png)

What a place to put it in. Thanks much for pointing this out!
